src/plugins/yambi/README.md

```diff
@@ -0,0 +1,87 @@
+- infos = Information about the yambi plugin is in keys below
+- infos/author = René Schwaiger <sanssecours@me.com>
+- infos/licence = BSD
+- infos/needs = directoryvalue yamlsmith
+- infos/provides = storage/yaml
+- infos/recommends =
+- infos/placements = getstorage
+- infos/status = maintained unittest shelltest preview experimental unfinished nodoc concept discouraged
```

Comment:
I absolutely agree with under promise but I think what your infos/status says: "Do not even think of using this". To say what you want to say, "experimental" alone should be okay, no "concept" or "discouraged" is needed.

Obviously the infos/status needs to be in-line with all the other YAML plugins because these plugins are are compared with each other.

Do you sometimes look at the output of `kdb info -v yaml`? It is important that the ordering there  makes sense.

Reply:
> I absolutely agree with under promise but I think what your infos/status says: "Do not even think of using this".

That works for me too :o).

> Do you sometimes look at the output of `kdb info -v yaml`?

Nope.

>  It is important that the ordering there makes sense.

The command `kdb info` does not have an option `-v` according to the output of `kdb info -v yaml` on my machine:

```
kdb: invalid option -- v
Sorry, I could not process the given options (see errors above)

Usage: kdb info <plugin name> [<clause name>]

Print information about a plugin.
Print out the information of a specific plugin's contract.

E.g.
Print out all information about dump plugin:
 kdb info dump

Print out the licence of the resolver plugin:
 kdb info resolver licence

-l --load                Load plugin even if system/elektra is available.
-H --help                Show the man page.
-p --profile <name>      Use a different profile for kdb configuration.
-V --version             Print version info.
-c --plugins-config <c>  Add a plugin configuration.
-C --color <when>       Print never/auto(default)/always colored output.
```

.